openmp/libomptarget/include/PluginManager.h

```diff
@@ -190,6 +190,11 @@ struct PluginManager {
   /// Add \p Flags to the user provided requirements.
   void addRequirements(int64_t Flags) { Requirements.addRequirements(Flags); }
 
+  bool getUseAutoZeroCopy() const { return UseAutoZeroCopy; }
```

Comment:
Just out of curiosity, why is it called auto zero copy instead of zero copy?

Reply:
zero-copy (or no-copy) is what is commonly called the support to requires unified_shared_memory. In that case, the user explicitly asks for the support, as opposed to the proposed support which turns on "automatically" if the system is an APU and XNACK (unified memory) support is enabled.